webgui/scripts/ga.js

```diff
@@ -0,0 +1,20 @@
+$(document).ready(function() {
+    $.ajax({
+        url: 'ga.txt',
+        dataType: 'text',
+        success: function (gaId) {
+            console.log('Google Analytics enabled: ' + gaId);
+            $.getScript('https://www.googletagmanager.com/gtag/js?id=' + gaId);
+
+            window.dataLayer = window.dataLayer || [];
+            function gtag(){dataLayer.push(arguments);}
+            gtag('js', new Date());
+            gtag('config', gaId);
+        },
+        statusCode: {
+            404: function() {
+                console.log('Google Analytics disabled.');
+            }
+        }
+    });
+});
```

Comment:
Please add the newline.

Reply:
> I think embedding Google Analytics (i.e. third-party, but also potentially invoking privacy issues!) stuff into the code needs to be looked over by the legal managerial team for approval.

CodeCompass is an open-source software, and this is a purely optional (by default disabled) feature, so I don't think any approval from Ericsson's legal managerial team is required. Legal approval *could be* required for CodeCompass servers running inside Ericsson with this feature enabled, but that is another topic.
I would like to emphasize that no Google Analytics code is executed when the feature is disabled, therefore there is no possible privacy issue in that case.

> The CodeCompass binary requiring _write_ access to the install directory is a too harsh requirement (for corporate deployments, anyways). [...]

Your technical suggestions are more concerning. To be honest, I wanted this be this a minimal effort feature, since it is not that important, but it would be nice to see some web traffic info e.g. on codecompass.net in the future.
But your concerns are valid, I will update this feature some time later and serve `/scripts/ga.txt` dynamically instead from the install folder.

Reply:
Not fixed. It could be that Git add needs to be forced, or hooks disabled at commit, so this goes through. (My local Git config is also iffy about newlines on brand new files...)

Reply:
Added it.